bin/send_metrics.py

```diff
@@ -0,0 +1,142 @@
+# pylint: skip-file
```

Comment:
Nit: Would be nice to add some docstrings on the top on the purpose of the script and how to use it. Basically how to run in case you are testing out a new use case and how to run in case you want to just send some metrics on existing use case.

Reply:
Updated, thank you!